Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.BeAssignableTo.cs

```diff
@@ -156,6 +157,25 @@ public void When_asserting_a_type_to_be_assignable_to_null_it_should_throw()
                 .WithParameterName("type");
         }
 
+        [Fact]
+        public void When_an_assertion_fails_on_BeAssignableTo_succeeding_message_should_be_included()
+        {
+            // Act
+            Action act = () =>
+            {
+                using var _ = new AssertionScope();
+                var item = string.Empty;
+                item.Should().BeAssignableTo<int>();
```

Comment:
This is the `ReferenceTypeAssertions.BeAssignableTo<T>` and not `TypeAssertions.BeAssignableTo<T>`.

Reply:
oops